README.md

```diff
@@ -25,13 +25,13 @@ Or install it yourself as:
 
 ```ruby
 require 'routemaster/client'
-client = Routemaster::Client.new(url: 'https://bus.example.com', uuid: 'demo')
+client = Routemaster::Client.new(url: 'https://bus.example.com', delivery_token: 'demo')
```

Comment:
More importantly, it was _wrong_. This one was the callback token; we now have `client_token` and `callback_token` :)